### PR TITLE
CRB-203: Update usage text. Rename orderId -> dealOrderId

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

File: ccclient/Program.cs [hunk 1]
```diff
@@ -97,14 +97,14 @@ static void Main(string[] args)
...
-                Console.WriteLine("show Transfer sighash|0 orderId");
+                Console.WriteLine("show Transfer sighash|0 dealOrderId");
                 Console.WriteLine("show CurrentLoans sighash|0");
                 Console.WriteLine("show LockedLoans sighash|0");
                 Console.WriteLine("show NewRepaymentOrders sighash|0");
                 Console.WriteLine("show CurrentRepaymentOrders sighash|0");
                 Console.WriteLine("creditcoin SendFunds amount sighash");
                 Console.WriteLine("creditcoin RegisterAddress blockchain address network");
-                Console.WriteLine("creditcoin RegisterTransfer gain orderId txId");
+                Console.WriteLine("creditcoin RegisterTransfer gain dealOrderId txId");
```
Comment (CAGS295) [inline ccclient/Program.cs]: What happens with the remaining `RegisterTransfer` commands from LOC 120-123? Consistency?

Comment (ghost) [reply ccclient/Program.cs]: I think @vladimir-gluwa intentionally called this orderID, since it can be a dealOrderID or a repaymentOrderID

Comment (atodorov) [reply ccclient/Program.cs]: The ticket explicitly says "in particular, orderId in "Usage" message should be dealOrderId;". I can also update linex 120-123 if this is the text we want to have .

**Comment by ghost** [comment]
I guess he changed his mind 🤷‍♂️

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by ghost — APPROVED** [review]

**Comment by CAGS295** [comment]
@atodorov For the Creditcoin transfer command. [this](https://github.com/gluwa/Creditcoin-Shared/blob/8ee6a009253449c40cf694f8b918d0f849050acd/cccore/Program.cs#L410) is the only reference that I could find and it looks like a transfer order which is neither `dealorderprefix` or `transferrepaymentprefix`. For the plugins it could either be `dealOrder | repaymentOrder` (e.g. [Bitcoin](https://github.com/gluwa/Creditcoin-Shared/blob/8ee6a009253449c40cf694f8b918d0f849050acd/plugins/bitcoin/Bitcoin.cs#L134))

**Review by ghost — APPROVED** [review]

**Review by seandeneen-gluwa — APPROVED** [review]

**Pull request merged**